### PR TITLE
[improve]: Adding additional instructions to readme.md

### PR DESCRIPTION
**Pull request by TRohit20**
Modified the current `readme.md` file by adding couple of additional instructions just to make it more clear and easier to setup Kubescape.io website locally making it easier to work with it and contribute if one wishes to. 

Thanks!

Screenshot 2023-09-19 at 11 04 30 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
+- Python installed on your local machine
+- pip 
 
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+### Setting Up
 
-### Production
+1. Clone the repository to your local machine.
+```bash
+git clone https://github.com/kubescape/website.git
```
Comment (craigbox) [inline README.md:26]: The repo isn't called `website`, it's called `kubescape.io`

I don't think we need to go to the level of teaching people how to check out the website in the readme.  Assume a basic level of background.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,54 @@
...
+pip install -r requirements.txt
+```
 
-The site is hosted with thanks to Netlify.  On each successful push to the repository, the site will be published by their bot.
\ No newline at end of file
+## Local Testing
+
+To build and test the website locally, follow these steps:
+
+1. Naviagte to project directory and run the command `mkdocs build`. The site will be built in the `site/` directory.
```
Comment (craigbox) [inline README.md:41]: `s/Naviagte/Navigate/`

You're already in the project directory from above.  Again, you don't need to lead people by the hand so much here.  Where we landed in the closed PR was perfect.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,54 @@
...
+
+## Production
+
+The site is currently hosted with thanks to Netlify. On each successful push to the repository, the site will be published by their bot.
+
+## Contributing
+
+We welcome contributions from the community. Please read our [contributing guide](https://github.com/kubescape/kubescape.io/blob/main/docs/project/contributing.md) for more information.
+
+## License
```
Comment (craigbox) [inline README.md:52]: The license is already prominently displayed on GitHub and a LICENSE file is included.

**Review by craigbox — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TRohit20) [reply README.md:26]: So would you suggest I remove the `setting up` section ? or Keep it short in just 2 points ?👇🏻

- Clone
- Install dependencies through requirements

File: README.md (hunk 2 above)
Comment (TRohit20) [reply README.md:41]: Noted. Will make the change

File: README.md (hunk 3 above)
Comment (TRohit20) [reply README.md:52]: Makes sense. Removing the License section then, Sound good?

File: README.md [hunk 4]
```diff
@@ -1,22 +1,43 @@
...
 
-## Building the site
+## Table of Contents
 
-### Local testing
+- [Getting Started](#getting-started)
+- [Local Testing](#local-testing)
+- [Production](#production)
+- [Contributing](#contributing)
+- [License](#license)
```
Comment (craigbox) [inline README.md:11]: You didn't remove the license.  And you really don't need a table of contents in a 10 line file.

File: README.md [hunk 5]
```diff
@@ -1,22 +1,43 @@
...
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+## Getting Started
 
-To build and test locally: 
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
```
Comment (craigbox) [inline README.md:16]: people expect that from a README, remove this section

File: README.md [hunk 6]
```diff
@@ -1,22 +1,43 @@
...
 
-* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker 
-* run `mkdocs serve`
+### Prerequisites
 
-## Local build
+- Python installed on your local machine
+- pip
```
Comment (craigbox) [inline README.md:20]: You don't need to say "on your local machine"?

File: README.md [hunk 7]
```diff
@@ -1,22 +1,43 @@
...
 
-## Local build
+- Python installed on your local machine
+- pip 
 
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+### Setting Up
 
-### Production
+1. Clone the repository to your local machine.
```
Comment (craigbox) [inline README.md:24]: this file is cloned to people's local checkout already, they can be reading it there.

File: README.md [hunk 8]
```diff
@@ -1,22 +1,43 @@
...
+```
 
-The site is hosted with thanks to Netlify.  On each successful push to the repository, the site will be published by their bot.
\ No newline at end of file
+## Local Testing
+
+To build and test the website locally, follow these steps:
+
+1. Run the command `mkdocs build`. The site will be built in the `site/` directory.
+2. After building the website locally, you can run the website locally using the command `mkdocs serve`.
```
Comment (craigbox) [inline README.md:35]: too wordy.  say "site", not "website", too. 

Was there a reason you thought the level of detail I had before was the wrong level?

**Review by craigbox — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (TRohit20) [reply README.md:35]: Nothing wrong but figured can modify a little to make it more clear.

**Review by craigbox — APPROVED** [review]
